design/baremetal-operator/firmware-image-update.md

```diff
@@ -0,0 +1,195 @@
+# Firmware Image Update
+
+## Status
+
+provisional
+
+## Summary
+
+Provide the ability to get and update vendor firmware version.
+
+## Motivation
+
+It desired to update vendor firmware before deploying the OS image of the host.
+
+### Goals
+
+- Agree on the format of the vendor neutral API and attributes

```

Comment:
'Goals' mean the goals of the feature, not the goals of this document.

I would say the goal is to have a consistent API where the vendor-specific parts are well-defined.